src/Protocol/Protocol_1_8.cpp

```diff
@@ -100,10 +100,10 @@ extern bool g_ShouldLogCommIn, g_ShouldLogCommOut;
 ////////////////////////////////////////////////////////////////////////////////
 // cProtocol_1_8_0:
 
-cProtocol_1_8_0::cProtocol_1_8_0(cClientHandle * a_Client, const AString & a_ServerAddress, UInt16 a_ServerPort, UInt32 a_State) :
+cProtocol_1_8_0::cProtocol_1_8_0(cClientHandle * a_Client, const AString & a_ServerAddress, UInt16 a_ServerPort, State a_State) :
 	Super(a_Client),
 	m_ServerAddress(a_ServerAddress),
-	m_ServerPort(a_ServerPort),
+	// Note: a_ServerPort is unused
```

Comment:
If it's unused, why not remove it from the constructor?

Reply:
Come to think of it I made all the constructors 1.9+ inherit so this change shouldn't need to touch every file.